Forge/src/main/resources/assets/placeableitems/lang/en_us.json

```diff
@@ -1,3 +1,6 @@
 {
-    "block.placeableitems.saddle_stand_block": "Saddle Stand"
+    "block.placeableitems.saddle_stand_block": "Saddle Stand",
+
+    "key.categories.placeableitems": "Placeable Items",
+    "key.placeableitems.place_special_item": "Place Special Item"
```

Comment:
I think this should read "Place Item". Also if you have other things assigned to Shift (such as crouch), does it mark it as a conflict in the UI?

Reply:
The reason I didn't go with `Place Item` is because Vanilla has one titled the same, though I guess if it's under its own category it shouldn't be much of an issue

Reply:
I think "Place Item" would be more desirable, even if the same name is used by MC

I'll do some tests tonight to see how the conflicts behave to make sure it's as intuitive as possible

Reply:
The only issue the conflicts have is red text for the keybinding in the control settings menu. Aside from that, it works flawlessly having it bound to the same key as any other. That aside, with the exception of Fabric. For some reason its keybind API is strange and `FabricKeyBinding.isPressed()` returns false when they're bound to the same key, though I worked around this by polling GLFW directly.

**TL;DR:** No gameplay difference. Works as expected.

Reply:
After consideration, perhaps we could append `(Hold)` to the keybind's name. `Place Item (Hold)`. This would at least be an indication of how it works

Reply:
That's a good idea. I'll clone this locally and want to try it out before approving, see if I can get a text as clear as possible so users don't swarm me with the same questions ;)

Reply:
Alright, after some testing, I think that adding "(Hold)" at the end may help users understand better.

It looks like there's a method inside `KeyBinding` called `conflicts`. Maybe it would be possible to override it and return false so it never appears red?

Reply:
As mentioned before, I'm just not sure that's the best approach. It seems slightly unsafe. Take for example, perhaps I have my tilde key assigned to both this keybind and that of inventory sorting. The configuration screen won't recognize it as a conflict and they'll function as intended, however I may accidentally organize my inventory while trying to place a block.

I could make that change and make it so it doesn't show it as conflicting, but in my personal opinion, I think it's better to show it as conflicting than not.

Alternatively, I could override `KeyBinding#conflicts()` and state that it won't conflict with the sneak key binding. I feel as though that may make the most sense.

Reply:
That makes sense. Alright nevermind then

Reply:
Would you mind adding the "(Hold)" at the end? Then we'll be good for final review